types/options.go

```diff
@@ -42,6 +42,7 @@ func init() {
 	defaultStoreOptions.GraphDriverName = ""
 
 	if _, err := os.Stat(defaultOverrideConfigFile); err == nil {
+		defaultConfigFile = defaultOverrideConfigFile
```

Comment:
Could you add a comment to this line?  I had to jump back two times to the commit message to understand why we need that change and am afraid of my garbage collector to forget so quickly.